src/ProxyClient/VarnishAdminMultiple.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace FOS\HttpCache\ProxyClient;
+
+class VarnishAdminMultiple
+{
+    public function __construct($servers)
+    {
+
+    }
+}
```

Comment:
Not sure yet whether we should support multiple servers in the `VarnishAdmin` class itself, or wrap them in a separate class such as this `VarnishAdminMultiple`.